tests/console/journalctl.pm

```diff
@@ -61,8 +66,11 @@ sub verify_journal {
     } elsif (defined($fss_key) && (script_run("egrep 'No sealing yet,.*of entries not sealed.' errs") == 0)) {
         die "Sealing is not working!\n";
         # Check for https://bugzilla.suse.com/show_bug.cgi?id=1178193, a race condition for `journalctl --verify`
-    } elsif ((script_run("grep 'File corruption detected' errs") == 0) && script_run("$cmd")) {
+        #    } elsif ((script_run("grep 'File corruption detected' errs") == 0)) {
```

Comment:
left over?

Reply:
yup, haven't pushed the latest change. thanks for letting me know